packages/react-day-picker/src/components/Navigation/Navigation.tsx

```diff
@@ -11,9 +11,9 @@ export interface NavigationProps {
   /** The month where the caption is displayed. */
   displayMonth: Date;
   /** The previous month. */
-  previousMonth?: Date;
+  previousMonth?: Date | undefined;
   /** The next month. */
-  nextMonth?: Date;
+  nextMonth?: Date | undefined;
```

Comment:
OK this change make sense: I can understand we are passing `undefined` to `Navigation` and here it should fail.